### ISSUE TITLE
Set Top Feature Window Border and Hotkeys Settings has bug

### ISSUE THREAD

**Issue by chinanoahli** [WindowTop/WindowTop-App#267]
I am running WindowTop as admin.
I set some windows to top (with `Highlight top most windows with red border` option enable),
and move the windows after set top.

And then the borders of the set top windows will not move with the windows,
but stay at the location that I setting the windows to top before.

The only way to solve it, is to cancel the set top windows, and move to the new place, and reset top windows again.
Or just disable `Highlight top most windows with red border` enable.

It looks like this:
![window top border bug](https://user-images.githubusercontent.com/10005128/196075460-f2e00fba-30c1-4d95-87fb-c84a3f0fa0d9.png)

**Comment by chinanoahli** [comment]
I am using Win10 LTSC 19044.1288

**Comment by chinanoahli** [comment]
And the Toolbar never show once for me, the `Integrate with the minimize button` option are not working, after enable it, noting changed.

**Comment by chinanoahli** [comment]
The HotKeys Settings has issue too:
After I set some hotkey to an input space, it will keep waiting for next input without auto stop read inputs, you can just continue to press orther hotkey, it will keep changing until the user click outside of the setting window, or click other tab of the setting window.

**Comment by gileli121** [comment]
@chinanoahli 
Can you tell if can find this window:
https://github.com/BiGilSoft/WindowTop/issues/151

If you see some blue window with red error text, it means that there was a fatal error.
When this happens, the behavior you saw is expected because the app is designed to suspend anything that it does, including managing the border (this is why they stop moving) and the toolbar (so you can't get the toolbar).
In this state, the app altogether suspended itself (it suspended all its main threads that run its code)

If you found this window, please send me the error in the window.
What version do you use?
How did you install it? Via Microsoft Store or regular installer, or is it the portable version?
Is there anything that you did before this issue? Or it happens right after running the app?

**Comment by chinanoahli** [comment]
I saw the error window once, but after I relaunch the WindowTop, it never shows again.
I was using the portable version WindowTop because LTSC has no Microsoft store feature.
This issue happened at the first time when I was using WindowTop, I remember that time I was trying to enable `Integrate with the minimize button`, but the app just frozen immediately. After about half a minute, the error window shows.
and then, I just relaunched the app.
